### PR TITLE
fix: fix adventure processing if there is more than one for a single …

### PR DESCRIPTION
**Pull request by maximilianvoss**
…slug

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by grubyak — APPROVED** [review]

File: src/components/AdventureDetail.jsx [hunk 1]
```diff
@@ -140,6 +140,10 @@ function getAdventure(data) {
 		// expect there only to be a single adventure in the array
 		if (data.adventureList.items.length === 1) {
 			return data.adventureList.items[0];
+		} else {
+			return data.adventureList.items.filter(item => {
+				return item._path.startsWith("/content/dam/wknd-shared/en");
+			})[0];
```
Comment (comanV) [inline src/components/AdventureDetail.jsx:146]: Could be simplified to `return data.adventureList.items.filter(item => item._path.startsWith("/content/dam/wknd-shared/en"))[0];`

Comment (hydralien) [reply src/components/AdventureDetail.jsx:146]: also could be find() and not filter, then it'd just return the first found item

Comment (hydralien) [reply src/components/AdventureDetail.jsx:146]: ...but that's just a comment, unimportant

**Review by hydralien — APPROVED** [review]

**Review by comanV — APPROVED** [review]

File: src/components/AdventureDetail.jsx (hunk 1 above)
Comment (maximilianvoss) [reply src/components/AdventureDetail.jsx]: I like the find... changed it :)

**Pull request merged**